lib/tyk/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.7'
+
+services:
+  tyk:
+    container_name: tyk
+    #build:
+    #  context: . # ${PWD}/lib/tyk
+    image: tykio/tyk-gateway:v2.9.3.1
+    networks:
+      - ${DOCKER_NET}
+      # - docker-compose-test-network
+    ports:
+      - "${TYK_SERVICE_PUBLIC_PORT}:8080"
+      # - "8080:8080"
+    deploy:
+      placement:
+        constraints:
+          - node.role == manager
+      restart_policy:
+        condition: on-failure
+        delay: 5s
+        max_attempts: 3
+        window: 120s
+    volumes:
+      - ./volumes/authMiddleware.js:/opt/tyk-gateway/middleware/authMiddleware.js
+      - ./volumes/oidcDistributedClaimsConduitMiddleware.js:/opt/tyk-gateway/middleware/oidcDistributedClaimsConduitMiddleware.js
+      - ./volumes/tyk.conf:/opt/tyk-gateway/tyk.conf
+      - ./volumes/virtualLogin.js:/opt/tyk-gateway/middleware/virtualLogin.js
+      - ./volumes/virtualLogout.js:/opt/tyk-gateway/middleware/virtualLogout.js
+      - ./volumes/virtualToken.js:/opt/tyk-gateway/middleware/virtualToken.js
+      - ./volumes/api_candig.json:/opt/tyk-gateway/apps/api_candig.json
+      - ./volumes/api_auth.json:/opt/tyk-gateway/apps/api_auth.json
+      - ./volumes/policies.json:/opt/tyk-gateway/policies/policies.json
+    depends_on:
+      - tyk-redis
+  tyk-redis:
+    container_name: tyk-redis
+    image: redis:4.0.14-alpine
+    networks:
+     - ${DOCKER_NET}
+    volumes:
+      - ./volumes/redis-data:/data
+
+networks:
+  bridge-net:
+    external: true
```

Comment:
this docker-compose does not conform to specs in lib/templates

Reply:
without this network tag, i am faced with errors stating the network doesn't exist.
any ideas?